source/CardGames.py

```diff
@@ -152,27 +152,24 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
 # NEW CODE BELOW
```

Comment:
Great job with the code! Make sure your next pull request name is more specific than "first commit" though